zulip/zulip/__init__.py

```diff
@@ -1076,6 +1076,21 @@ def delete_stream(self, stream_id):
             method='DELETE',
         )
 
+    def get_member(self, user_id, request=None):
+        # type: (int, Optional[Dict[str, Any]]) -> Dict[str, Any]
+
+        '''
+            Example usage:
+
+            >>> client.get_member(8,{'include_custom_profile_fields': True})
```

Comment:
I'd put a space after the `,` here, as that's standard Python style.  Otherwise lgtm.